icon_exporter.rb

```diff
@@ -0,0 +1,57 @@
+#! /usr/bin/env ruby
```

Comment:
@dcocchia This will probably come in handy in the future, where should we put it?